testing/test_aux.out

```diff
@@ -11,12 +11,26 @@ RESULT info protocol.bacnet.version Protocol version: 1
 RESULT pass protocol.bacnet.pic
 RESULT fail connection.mac_oui Manufacturer prefix not found!
 RESULT pass connection.mac_oui
-RESULT skip security.tls.v3
-RESULT skip security.x509
-RESULT fail security.tls.v3
-RESULT fail security.x509
-RESULT pass security.tls.v3
-RESULT pass security.x509
+RESULT fail security.tls.v1
+RESULT fail security.tls.v1.x509
+RESULT fail security.tls.v1_2
+RESULT fail security.tls.v1_2.x509
+RESULT fail security.tls.v1_3
+RESULT fail security.tls.v1_3.x509
+RESULT fail security.tls.v1_2
+RESULT fail security.tls.v1_2.x509
+RESULT fail security.tls.v1_3
+RESULT fail security.tls.v1_3.x509
+RESULT pass security.tls.v1
+RESULT pass security.tls.v1.x509
+RESULT pass security.tls.v1_2
+RESULT pass security.tls.v1_2.x509
+RESULT fail security.tls.v1_3
```

Comment:
there needs to be something other than just "fail" for the tests, otherwise there's no point in including them since they have no purpose (from a software perspective). How do you know these are working properly?

Shouldn't there also be a skip condition where there is to viable port to confirm TLS on?

Reply:
The faux devices currently available do not have TLS 1.3 support.  I can't get passing results unless these devices are updated to support it.  All TLS version testing follow the exact same pattern but just defines the appropriate accepted TLS version for each test.

Reply:
Sill missing the pass variants for v1_3... the java-11 version of the faux device is available so should be suitable for adding in 1.3